lib/src/notification/notification.component.scss

```diff
@@ -0,0 +1,286 @@
+
+$small: 12px;
+
+@mixin apply-theme($bg-color, $close-hover) {
+    .content-wrapper {
+        background-color: $bg-color;
+
+        &.clickable {
+            &:hover {
+                background-color: lighten($bg-color, 8%);
+            }
+
+            &:active {
+                background-color: $bg-color;
+            }
+        }
+
+        .actions-wrapper {
+            .action-wrapper {
+                .notification-action {
+                    &:hover {
+                        color: $bg-color;
+                    }
+                }
+            }
+        }
+    }
+
+    .dismiss-btn {
+        svg {
+            &:hover {
+                fill: $close-hover;
+            }
+        }
+    }
+}
+
+.custom-notification {
+    position: fixed;
+    display: block;
+    z-index: 9999;
+    opacity: 0;
+    will-change: transform, opacity;
+    transition: transform 600ms cubic-bezier(0.65, -0.15, 0.47, 1.16), opacity 800ms ease-in-out;
+
+    .content-wrapper {
+        min-height: 30px;
+        position: relative;
+        box-shadow: 0 4px 4px -2px rgba(0, 0, 0, 0.25), 3px 4px 15px rgba(0, 0, 0, 0.5);
+        padding: 8px 13px;
+        color: $white;
+
+        &.clickable {
+            cursor: pointer;
+        }
+
+        .notification-title {
+            font-weight: 500;
+            padding-right: 10px;
+            user-select: none;
+            display: inline-block;
+        }
+
+        .notification-message {
+            font-size: $small;
+            padding-right: 18px;
+            user-select: none;
+        }
+    }
+
+    .dismiss-btn {
+        position: absolute;
+        top: 12px;
+        right: 8px;
+        cursor: pointer;
+        will-change: top, right;
+
+        svg {
+            fill: $white;
+            position: relative;
+            display: block;
+            width: 16px;
+            height: 16px;
+        }
+    }
+
+    &.style-slide-in {
+        width: 300px;
+
+        &.bottom-left {
+            transform: translateX(-350px);
+            left: 20px;
+            bottom: 15px;
+        }
+
+        &.bottom-right {
+            transform: translateX(350px);
+            right: 20px;
+            bottom: 15px;
+        }
+
+        &.top-left {
+            transform: translateX(-350px);
+            left: 20px;
+            top: 15px;
+        }
+
+        &.top-right {
+            transform: translateX(350px);
+            right: 20px;
+            top: 15px;
+        }
+
+        &.open {
+            transform: translateX(0);
+            opacity: 1;
+        }
+
+        .content-wrapper {
+            border-radius: 4px;
+            will-change: box-shadow, top, left;
+            transition: box-shadow 80ms linear, top 100ms ease, left 100ms ease, background-color 200ms ease;
+
+            &.clickable {
+                &:active {
+                    box-shadow: 0 0 0 0 rgba(0, 0, 0, 0), 1px 2px 11px rgba(0, 0, 0, 0.5);
```

Comment:
move this to a variable

Reply:
While the styling changes might make sense, it would represent a sharp decline from it resemblance with our current react notification component stylings. If they are necessary then perhaps we should be moving the same to react component too. The styling file here is exact replica of what we have in react component.

Reply:
I will be moving it to a variable as soon as I get to it during my refactoring